apps/studio/components/interfaces/Database/Schemas/SchemaGraph.tsx

```diff
@@ -157,7 +157,7 @@ export const SchemaGraph = () => {
               deletable: false,
               style: {
                 stroke: 'hsl(var(--border-stronger))',
-                strokeWidth: 0.5,
+                strokeWidth: 1,
```

Comment:
unrelated change here but its just to bring up the contrast of the schema graph lines for foreign keys (there were some feedback about this)